sui-execution/latest/sui-adapter/src/temporary_store.rs

```diff
@@ -48,17 +46,10 @@ pub struct TemporaryStore<'backing> {
     /// The version to assign to all objects written by the transaction using this store.
     lamport_timestamp: SequenceNumber,
     mutable_input_refs: BTreeMap<ObjectID, (VersionDigest, Owner)>, // Inputs that are mutable
-    // When an object is being written, we need to ensure that a few invariants hold.
-    // It's critical that we always call write_object to update `written`, instead of writing
-    // into written directly.
-    written: BTreeMap<ObjectID, (Object, WriteKind)>, // Objects written
-    /// Objects actively deleted.
-    deleted: BTreeMap<ObjectID, DeleteKindWithOldVersion>,
+    execution_results: ExecutionResultsV2,
     /// Child objects loaded during dynamic field opers
     /// Currently onply populated for full nodes, not for validators
-    loaded_child_objects: BTreeMap<ObjectID, LoadedChildObjectMetadata>,
-    /// Ordered sequence of events emitted by execution
-    events: Vec<Event>,
+    loaded_child_objects: BTreeMap<ObjectID, LoadedRuntimeObjectMetadata>,
```

Comment:
it might be a bit late to bring this up, but I have a nagging feeling that it might be useful to preserve ordering of reads/writes, rather than sorting by ID. It gives us strictly more information without costing any space, and now is probably the only opportunity we'd have to do this. Thoughts?

Reply:
We are not committing to loaded child objects in effects v2 yet. So it doesn't matter that much here.